recipe/meta.yaml

```diff
@@ -24,7 +24,7 @@ test:
   commands:
     - test -d ${PREFIX}/include/dlpack  # [unix]
     - test -f ${PREFIX}/include/dlpack/dlpack.h  # [unix]
-    - test -f ${PREFIX}/bin/mock  # [unix]
```

Comment:
Do we know why `mock` was being installed before? AIUI this is just a test executable that doesn't really do anything. Though maybe I'm missing something about why it was needed

Reply:
I looked at the git blame and it was included in the initial commit so I couldn't find any discussion or reasoning. In the dlpack repo it seems to be just to ensure that things compile as expected. For that reason, I left it to continue building here just in case of a compiler hiccup, but it doesn't get shipped as part of the package.

Reply:
Looks like this goes back to PR ( https://github.com/conda-forge/staged-recipes/pull/8302 )

@marcelotrevisani, do you have thoughts on this? Is this ok to drop for you?

Reply:
@marcelotrevisani gentle ping. I plan to merge this in the next couple days if I don't hear otherwise.

Reply:
We can always follow up here if a change is needed